addons/sourcemod/scripting/shop/item_manager.sp

```diff
@@ -1514,7 +1517,7 @@ bool ItemManager_GetItemDisplay(int item_id, int source_client, char[] buffer, i
 	return true;
 }
 
-bool ItemManager_FillItemsOfCategory(Menu menu, int client, int source_client, int category_id, bool inventory = false, bool showAll = false)
+bool ItemManager_FillItemsOfCategory(Menu menu = null, int client, int source_client, int category_id, bool inventory = false, bool showAll = false, bool shouldTest = false)
 {
```

Comment:
In what case menu there got null value?

Reply:
To check for the presence of items in the category, taking into account Should, it was required to receive a response from [this place,](https://github.com/FD-Forks/Shop-Core/pull/85/files/acf6f7e0ceecdd311d26c6856ffdfb9e16940851#diff-df645dc81cf2426987ac7db1517698fa108117e9c2a54dc07827c56544466a48R1417) and in order not to add items to the menu, I made it null and check for this below. Checking items is only called in one place, this should not greatly affect the work of other functions

Reply:
I don't like how it works now, mainly on code part which makes you to reuse ItemManager_FillItemsOfCategory 2 times instead of once.